lib/kinesis_client/stream/app_state/ecto/shard_leases.ex

```diff
@@ -12,11 +18,18 @@ defmodule KinesisClient.Stream.AppState.Ecto.ShardLeases do
     end
   end
 
+  def get_shard_lease(_params, _repo), do: {:error, :missing_required_fields}
+
   @spec get_shard_lease_by_id(String.t(), Ecto.Repo.t()) ::
-          {:error, :not_found} | {:ok, ShardLease.t()}
+          {:error, :not_found} | {:ok, list(ShardLease.t())}
```

Comment:
@tkusumo can this function be removed?   Looks like it's only used by tests.

Reply:
Function removed 👍